other-formats/README.md

```diff
@@ -0,0 +1,34 @@
+# Alternative Formats
+This directory contains `convert.sh`, which will convert the contents of the book from
+Markdown into a variety of other formats.
+
+Supported formats are
+
+ - LaTeX
+ - HTML
+ - Man page
+
+## Running the Script
+
+Very simple:
+
+    $ ./convert.sh
+
+This will convert the markdown files from book into all of the supported formats.
```

Comment:
Can we have this convert only to what the user wants? I.e. add an option for which formats you'd like to generate.